vue-components/package-lock.json

```diff
@@ -62,6 +62,7 @@
 				"postcss": "^7.0.35",
 				"postcss-dir-pseudo-class": "^5.0.0",
 				"postcss-inline-svg": "^4.1.0",
+				"postcss-loader": "^6.2.1",
```

Comment:
I’m confused by this change (and the webpack removal below) – isn’t this part of the package-lock.json supposed to be the same as the (dev)dependencies of package.json? But the package.json isn’t touched in this commit.

Reply:
yes, npm is incredibly weird. as I understand it, `package-lock.json` is only updated iff `node_modules` is updated. So if something is changed to `package.json` and that change did not result in a change being need in `node_modules`, then `package-lock.json` is also not updated. Even though some dependencies might have nominally changed in the top-level `package.json`. I'm not yet 100% sure for how this plays with `npm install --package-lock.only`.